CI failed to publish

Failed to publish to npmjs for tag `v0.7.0-beta.10`:

https://app.circleci.com/pipelines/github/lightbend/akkaserverless-javascript-sdk/157/workflows/096235a2-e7db-400c-9b9e-e5582b8b05e9/jobs/621

```
npm ERR! code E401
npm ERR! Unable to authenticate, need: Basic realm="https://npm.cloudsmith.io/lightbend/akkaserverless/",service="npm.cloudsmith.io"
```

Because we set `@lightbend` to fetch from cloudsmith for forked dependencies that we've created:

https://github.com/lightbend/akkaserverless-javascript-sdk/blob/0aa8daa99a8bc201116dc5da18d4350c68f8c7a4/sdk/.npmrc#L2

--- a/sdk/package.json
+++ b/sdk/package.json
@@ -87,5 +87,8 @@
     "prepare": "bin/prepare.sh",
     "postinstall": "node bin/download-protoc.js",
     "prepublishOnly": "npm version --no-git-tag-version $(bin/version.sh)"
+  },
+  "publishConfig": {
+    "@lightbend:registry": "https://registry.npmjs.org"
   }
 }

